amundsen_application/static/js/components/common/Breadcrumb/index.tsx

```diff
@@ -13,51 +12,35 @@ export interface OwnProps {
   text?: string;
 }
 
-export interface StateFromProps {
-  searchTerm: string;
-}
-
 export interface MapDispatchToProps {
   loadPreviousSearch: () => LoadPreviousSearchRequest;
 }
 
-export type BreadcrumbProps = OwnProps & StateFromProps & MapDispatchToProps;
+export type BreadcrumbProps = OwnProps & MapDispatchToProps;
 
 export const Breadcrumb: React.SFC<BreadcrumbProps> = (props) => {
-  let path = props.path;
-  let text = props.text;
-  if (!path && !text) {
-    path = '/';
-    text = 'Home';
-    if (props.searchTerm) {
-      return (
-        <div className="amundsen-breadcrumb">
-          <a onClick={ props.loadPreviousSearch } className='btn btn-flat-icon title-3'>
-            <img className='icon icon-left'/>
-            <span>Search Results</span>
-          </a>
-        </div>
-      );
-    }
+  const { path, text } = props;
+  if (path !== undefined && text !== undefined) {
```

Comment:
1. Nit: With this change we no longer have a use case for a `Breadcrumb` with text, and I also haven't seen it in any of our future designs anymore, so I'd be up for simplifying the component to just take `path`.
2. Nit: Do we not want to protect against `path` being an empty string? I don't feel strongly either way because I think the onus is on the developer to use the component props properly.

Reply:
I did initially remove the path and text props, but added them back in. There is one usage in the 404 page but if we decide that is not useful, we can cleanup here.

Reply:
Let's leave as is then and not increase scope.